spec/interactors/add_playoffs_to_tournament_interactor_spec.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-RSpec.describe AddPlayoffsToTournament do
+RSpec.describe AddPlayoffsToTournament, type: :interactor do
```

Comment:
what's the purpose of this?

Reply:
That is what they autogenerate with, so I thought I would put it there for the sake of completion